src/w3d/renderer/dx8wrapper.cpp

```diff
@@ -2000,17 +2000,25 @@ void DX8Wrapper::Set_Render_Target(w3dsurface_t render_target, bool use_default_
             DX8CALL(SetRenderTarget(s_defaultRenderTarget, s_defaultDepthBuffer));
             s_defaultRenderTarget->Release();
             s_defaultRenderTarget = nullptr;
-            if (s_defaultDepthBuffer) {
+
+            if (s_defaultDepthBuffer != nullptr) {
```

Comment:
Is this meant to be inside scope of `if (s_defaultRenderTarget != nullptr)` condition ?